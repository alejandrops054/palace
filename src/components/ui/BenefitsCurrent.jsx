import React from "react";

//title: string
//price: int
//totalprice: int

const BenefitsCurrent = ({ title="", price, totalprice }) => {
    return (
        <h3> {title} <span id="current-amount">${price}</span>/<span id="total-amount">${totalprice}</span></h3>
    );
};

export default BenefitsCurrent;