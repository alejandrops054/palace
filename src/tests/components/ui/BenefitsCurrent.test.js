/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import BenefitsCurrent from '../../../components/ui/BenefitsCurrent'

describe('Pueba en <BenefitsCurrent/>', () => {
    test('Debe renderizar el componente con titulo, precio y precio total', () => {
        const titulo = 'Titulo BenefitsCurrent';
        const price = '500'
        const totalPrice = '5000'
        const wrapper = shallow(
            <BenefitsCurrent
                title={titulo}
                price={price}
                totalPrice={totalPrice}
            />
        );

        expect(wrapper).toMatchSnapshot();
    })

})