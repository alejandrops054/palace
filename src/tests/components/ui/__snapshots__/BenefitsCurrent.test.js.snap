// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pueba en <BenefitsCurrent/> Debe renderizar el componente con titulo, precio y precio total 1`] = `
<h3>
   
  Titulo BenefitsCurrent
   
  <span
    id="current-amount"
  >
    $
    500
  </span>
  /
  <span
    id="total-amount"
  >
    $
  </span>
</h3>
`;
